src/app/components/hris/employees/new-employee/new-employee.component.ts

```diff
@@ -192,7 +192,7 @@ export class NewEmployeeComponent implements OnInit {
     reportingLine: new FormControl<EmployeeProfile | null>(null),
     highestQualication: new FormControl<string>(''),
     disability: new FormControl<boolean | null>(false, [Validators.required]),
-    disabilityNotes: new FormControl<string>(''),
+    disabilityNotes: new FormControl<string>('', [Validators.required]),
```

Comment:
Disability notes are only required if the boolean value for disability is yes

Reply:
The disabilityNotes only starts to display when the value 'yes' is selected otherwise it is hidden and not visible to the user